client/reader/stream/empty-search-rec.jsx

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
this file is copy-pasted from search-stream with no changes because now it is used in post-lifecycle instead of from within the search-stream.  In a followup PR I plan on consolidating RecommendationBlock + this